src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -1095,15 +1095,21 @@ def get_lvs(fields=LV_FIELDS, filters='', tags=None):
     return [Volume(**lv_report) for lv_report in lvs_report]
 
 
-def get_first_lv(fields=LV_FIELDS, filters=None, tags=None):
+def get_single_lv(fields=LV_FIELDS, filters=None, tags=None):
     """
     Wrapper of get_lv meant to be a convenience method to avoid the phrase::
         lvs = get_lvs()
         if len(lvs) >= 1:
             lv = lvs[0]
     """
     lvs = get_lvs(fields=fields, filters=filters, tags=tags)
-    return lvs[0] if len(lvs) > 0 else []
+
+    if len(lvs) == 0:
+        raise RuntimeError('Filters {} did not match any lv present on this host.'.format(filters))
+    if len(lvs) > 1:
+        raise RuntimeError('Filters {} matched more than 1 LV present on this host.'.format(filters))
```

Comment:
`filters` are `dict`, they need to be converted to `str` first. Also, wrapping this at 80 would be nice.
```suggestion
    if len(lvs) == 0:
        raise RuntimeError('Filters {} did not match any lv present on this host.'.format(str(filters)))
    if len(lvs) > 1:
        raise RuntimeError('Filters {} matched more than 1 LV present on this host.'.format(str(filters)))
```